Signup/Signup.js

```diff
@@ -1,13 +1,12 @@
-const linkToPortfolioPage = '../Portfolio/Portfolio.html';
+const LinkToPortfolioPage = '../Portfolio/Portfolio.html';
```

Comment:
use ALL_CAPS for constants. In the previous comment i meant ALL_CAPS and mistakenly written pascal case

Reply:
Sure I will. Please let me know when you are done commenting

Reply:
@Asim-Khaleel review the new comments and fix all

Reply:
@sikandar-ramzan I have resolve the comments. Please review them